README.md

```diff
@@ -10,3 +10,67 @@ The Mattermost operator is in alpha, data loss might occur.
 Mattermost is a scalale, open source collaboration tool. It's written in Golang and React.
 
 The idea of this project is to build the Kubernetes Operator for Mattermost to mak more easy for Kubernetes users deploy and manage your Mattermost instance.
+
+
+## Getting Started
+
+### Prerequisites
+
+- Kubernetes cluster 1.8.0+.
+- `kubectl` configured to refer to relevant Kubernetes cluster.
+
+### Create Operator and related resources
+
+To start Mattermost-Operator, we need to install the dependencies first.
+
+#### MySQL-Operator
+To install MySQL-Operator apply the manifests that you can find in the `docs` folder
+
+```
+$ kubectl create ns mysql-operator
+$ kubectl apply -n mysql-operator -f https://github.com/mattertmost/mattermost-operator/blob/master/docs/mysql-operator/mysql-operator.yaml?raw=true
+```
+
+#### Mattermost-Operator
+After the dependencies installed we need to deploy Mattermost-operator
+Apply the manifests in the `docs` folder as well
+
+```
+$ kubectl create ns mattermost-operator
+$ kubectl apply -n mattermost-operator -f https://github.com/mattertmost/mattermost-operator/blob/master/docs/mattermost-operator/mattermost-operator.yaml?raw=true
```

Comment:
This URL is also not valid.

Reply:
not valid because we did not merge the this pr :)

Reply:
I'm pretty sure there is at least one typo in here as well. :)
```suggestion
$ kubectl apply -n mattermost-operator -f https://github.com/mattermost/mattermost-operator/blob/master/docs/mattermost-operator/mattermost-operator.yaml?raw=true
```